File: Day_1/Part_2.py
with open('Day 1/input.txt', 'r') as input:
    data = [int(line.strip()) for line in input]

for index_i, i in enumerate(data):
    for index_j, j in enumerate(data[index_i+1:]):
        for k in data[index_j+1:]:
            if i + j + k == 2020:
                print(i, j, k, i*j*k)

File: Day_1/Part_2.c

#include <stdio.h>
#include <stdlib.h>

int main()
{

    FILE *file;
    file = fopen("input.txt", "r");

    //read file into array
    int data[200];
    int i;

    if (file == NULL)
    {
        printf("Error Reading File\n");
        exit(0);
    }

    for (i = 0; i < 200; i++)
    {
        fscanf(file, "%d,", &data[i]);
    }

    for (i = 0; i < 199; i++)
    {
        for (int j = i + 1; j < 200; j++)
        {
            for (int k = j + 1; k <= 200; k++)
            {
                if (data[i] + data[j] + data[k] == 2020)
                {
                    printf("%d * %d * %d = %d", data[i], data[j], data[k], data[i] * data[j] * data[k]);
                }
            }
        }
    }

    fclose(file);

    return 0;
}
